core/files/css/base.php

```diff
@@ -762,6 +763,11 @@ private function get_selector_global_value( $control, $global_key ) {
 
 		if ( ! empty( $control['groupType'] ) ) {
 			$property_name = str_replace( [ $control['groupPrefix'], '_tablet', '_mobile' ], '', $control['name'] );
+
+			if ( empty( $data['value'][ Global_Typography::TYPOGRAPHY_GROUP_PREFIX . $property_name ] ) ) {
```

Comment:
It doesn't cover `multiple` controls like `font-size`.

Reply:
The data manager should return only changed values, without defaults

Reply:
It did it. If I understand correctly you only meant to check if `isset`?

Reply:
```suggestion
			if ( ! isset( $data['value'][ Global_Typography::TYPOGRAPHY_GROUP_PREFIX . $property_name ] ) ) {
```